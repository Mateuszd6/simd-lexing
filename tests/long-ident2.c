long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long

int DonaudampfschiffahrtselektrizitatenhauptbetriebswerkbauunterbeamtengesellschaftDonaudampfschifffahrtsgesellschaftskapitanRechtsschutzversicherungsgesellschaften()
{ return 0; }
