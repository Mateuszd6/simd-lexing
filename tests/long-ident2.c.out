Parsed: 10 lines, 99 ids, 0 strings, 0 chars, 1 ints, 0 hex,   0 floats,    0 //s, 0 /**/s,   0 #foo
